lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -251,6 +251,96 @@ def remove_testers(project_number, emails)
+        connection.post(add_tester_group_url(project_number), payload.to_json) do |request|
+          request.params["groupId"] = group_alias
+          request.headers[AUTHORIZATION] = "Bearer " + @auth_token
+          request.headers[CONTENT_TYPE] = APPLICATION_JSON
+          request.headers[CLIENT_VERSION] = client_version_header_value
+        end
+      rescue Faraday::BadRequestError
+        UI.user_error!(ErrorMessage::INVALID_TESTER_GROUP_NAME)
+      rescue Faraday::ResourceNotFound
+        UI.user_error!(ErrorMessage::INVALID_PROJECT)
+      rescue Faraday::ConflictError
+        UI.important("Tester group #{group_alias} already exists.")
+      rescue Faraday::ClientError => e
+        raise e
+      end
+
+      # Add testers to group
+      #
+      # args
+      #   project_number - Firebase project number
+      #   group_alias - Alias of the tester group
+      #   emails - An array of emails to be added to the group.
+      #            A maximum of 1000 testers can be added at a time, if creating missing testers is enabled.
+      #   create_missing_testers - If true, missing testers will be created and added to the group.
+      #
+      def add_testers_to_group(project_number, group_alias, emails, create_missing_testers = true)
+        payload = { emails: emails,
+                    createMissingTesters: create_missing_testers }
+        UI.message("Adding testers to group URL: #{add_testers_to_group_url(project_number, group_alias)}")
+        connection.post(add_testers_to_group_url(project_number, group_alias), payload.to_json) do |request|
+          request.headers[AUTHORIZATION] = "Bearer " + @auth_token
+          request.headers[CONTENT_TYPE] = APPLICATION_JSON
+          request.headers[CLIENT_VERSION] = client_version_header_value
+        end
+      rescue Faraday::BadRequestError
+        UI.user_error!(ErrorMessage::INVALID_EMAIL_ADDRESS)
+      rescue Faraday::ResourceNotFound
+        UI.user_error!(ErrorMessage::INVALID_TESTER_GROUP)
+      rescue Faraday::ClientError => e
+        raise e
+      end
+
+      # Remove testers from group
+      #
+      # args
+      #   project_number - Firebase project number
+      #   group_alias - Alias of the tester group
+      #   emails - An array of emails to be removed from the group.
+      #
+      def remove_testers_from_group(project_number, group_alias, emails)
+        payload = { emails: emails }
+        connection.post(remove_testers_from_group_url(project_number, group_alias), payload.to_json) do |request|
+          request.headers[AUTHORIZATION] = "Bearer " + @auth_token
+          request.headers[CONTENT_TYPE] = APPLICATION_JSON
+          request.headers[CLIENT_VERSION] = client_version_header_value
+        end
+      rescue Faraday::BadRequestError
+        UI.user_error!(ErrorMessage::INVALID_EMAIL_ADDRESS)
+      rescue Faraday::ResourceNotFound
+        UI.user_error!(ErrorMessage::INVALID_PROJECT)
```

Comment:
This should print `ErrorMessage::INVALID_TESTER_GROUP` since the group is the primary resource of this call.